### PR TITLE
Use sbrscrape package to get odds data from various sportsbooks

### PR DESCRIPTION
**Pull request by nkgilley**
pass in `--odds=draftkings` to specify a sportsbook to scrape from.

options include: fanduel, draftkings, betmgm, pointsbet, caesars, wynn, bet_rivers_ny

If anyone is interested I have also created a small Flask app to display the output from this tool in the browser.
https://github.com/nkgilley/flask-nba-ml

image

### CODE REVIEW

**Comment by kyleskom** [comment]
This looks amazing. Can't wait to test it soon.

File: main.py [hunk 1]
```diff
@@ -75,5 +95,6 @@ def main():
     parser.add_argument('-xgb', action='store_true', help='Run with XGBoost Model')
     parser.add_argument('-nn', action='store_true', help='Run with Neural Network Model')
     parser.add_argument('-A', action='store_true', help='Run all Models')
+    parser.add_argument('--odds', default='fanduel', help='Sportsbook to fetch from. (fanduel, draftkings, betmgm, pointsbet, caesars, wynn, bet_rivers_ny')
```
Comment (kyleskom) [inline main.py:98]: Can we change this to just `'-odds'` to match format.

**Review by kyleskom — CHANGES_REQUESTED** [review]

**Comment by kyleskom** [comment]
Can you also update to readme / have a doc for this usage. Looks amazing and tested, works as expected.

File: main.py [hunk 2]
```diff
@@ -46,11 +57,20 @@ def createTodaysGames(games, df):
...
 
 def main():
+    odds = None
     data = get_todays_games_json(todays_games_url)
     games = create_todays_games(data)
     data = get_json_data(data_url)
     df = to_data_frame(data)
-    data, todays_games_uo, frame_ml, home_team_odds, away_team_odds = createTodaysGames(games, df)
+    if args.odds:
+        print("--------------Automatic odds data scraping-------------")
```
Comment (kyleskom) [inline main.py:66]: I think we should remove this print statement. One thing thats not needed but maybe a nice to have would be a pretty print of the odds for the games.

**Comment by kyleskom** [comment]
I also really like the front end you have built with flask. Is there anyway we can merge that into this main repo as well?

**Comment by kyleskom** [comment]
One last thing. I think having there be a way to manually enter as well would be good. Leaving the automatic odds as default but a new param to enter manually.

File: main.py (hunk 1 above)
Comment (nkgilley) [reply main.py]: done

File: main.py (hunk 2 above)
Comment (nkgilley) [reply main.py]: done

**Comment by nkgilley** [comment]
> One last thing. I think having there be a way to manually enter as well would be good. Leaving the automatic odds as default but a new param to enter manually.

removed the default so now if `-odds` is not given you can enter manually

**Comment by nkgilley** [comment]
> I also really like the front end you have built with flask. Is there anyway we can merge that into this main repo as well?

done

**Comment by nkgilley** [comment]
I went ahead and included the flask app.  Right now it displays data from fanduel, draftkings, and betmgm.  It will also cache the results for 5 minutes before fetching new data and running the model again.

**Comment by kyleskom** [comment]
Man thank you so much. This PR has many things that I always wanted to add but never had time. Not only that it's done exceptionally well. Kudos.

**Review by kyleskom — APPROVED** [review]

**Comment by kyleskom** [comment]
Seeing some issues, should be 227
image

**Comment by kyleskom** [comment]
Tracking and future discussions. #55

**Pull request merged**